src/integration-test/java/org/web3j/quorum/ExtensionRequestTest.java

```diff
@@ -105,4 +105,24 @@ public void testExtensionStatus() throws Exception {
         verifyResult(
                 "{\"jsonrpc\":\"2.0\",\"method\":\"quorumExtension_getExtensionStatus\",\"params\":[\"managementContractAddress\"],\"id\":0}");
     }
+
+    @Test
+    public void testquorumExtensionGenerateExtensionApprovalUuid() throws Exception {
```

Comment:
change function name for camel case - "testQuorumExt..."

Reply:
@NickSneo changed to camel case. Thanks